help2man/help2man.inc

```diff
@@ -0,0 +1,9 @@
+# -*- mode:python; -*-
+RECIPE_TYPES = "machine native"
```

Comment:
Hehe, let's just change this to RECIPE_TYPES = "native" then :)  That way, we don't have to waste buildbot time building it for all machine archs.